pkg/cmd/root/help_topic.go

```diff
@@ -128,6 +128,7 @@ func NewHelpTopic(topic string) *cobra.Command {
 		Use:    topic,
 		Short:  HelpTopics[topic]["short"],
 		Long:   HelpTopics[topic]["long"],
+		Example: HelpTopics[topic]["example"]
```

Comment:
Needs trailing comma & `go fmt`

Reply:
That's what I get for using vscode.dev 😅 .

Fixed.